chatkit/build.gradle

```diff
@@ -33,6 +38,8 @@ dependencies {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
-    api 'com.pusher:pusher-platform:0.0.1-SNAPSHOT' //Remote pusher platform
     api 'org.jetbrains.kotlinx:kotlinx-coroutines-core:0.21.2'
+//    api 'com.pusher:pusher-platform:0.0.1-SNAPSHOT' //Remote pusher platform
```

Comment:
This bit really just needs to be uncommented and the line below removed. But we have to make sure we release the platform lib first and update the snapshot.